drivers/scheduler/k8s/k8s.go

```diff
@@ -2841,6 +2834,109 @@ func (k *K8s) destroyBackupObjects(
 	return nil
 }
 
+// EnableVolumeExpansion creates autopilot rules for given context
+func (k *K8s) EnableVolumeExpansion(ctx *scheduler.Context) error {
```

Comment:
can we do this EnableVolumeExpansion logic at Schedule time itself? In other words, do we need a new Interface and an explicit call from test to enable this.